ygen/codegen.go

```diff
@@ -341,15 +344,18 @@ func findMappableEntities(e *yang.Entry, dirs map[string]*yang.Entry, enums map[
 				if gch.IsContainer() || gch.IsList() {
 					dirs[fmt.Sprintf("%s/%s", ch.Parent.Path(), gch.Name)] = gch
 				}
-				errs = util.AppendErrs(errs, findMappableEntities(gch, dirs, enums, excludeModules, compressPaths, modules))
+				errs = util.AppendErrs(errs, findMappableEntities(gch, dirs, enums, excludeModules, compressPaths, ignoreUnsupportedStatements, modules))
 			}
 		case ch.IsContainer(), ch.IsList():
 			dirs[ch.Path()] = ch
 			// Recurse down the tree.
-			errs = util.AppendErrs(errs, findMappableEntities(ch, dirs, enums, excludeModules, compressPaths, modules))
+			errs = util.AppendErrs(errs, findMappableEntities(ch, dirs, enums, excludeModules, compressPaths, ignoreUnsupportedStatements, modules))
 		case ch.Kind == yang.AnyDataEntry:
 			continue
 		default:
+			if ignoreUnsupportedStatements {
```

Comment:
Can we log something here about what we're skipping? I worry that since `ignoreUnsupported` is quite generic then it becomes difficult to understand what is /not/ included in the code based on this.

Reply:
```
I0725 09:56:07.350343  442601 codegen.go:358] Unsupported statement (Notification) ignored: /openconfig-simple/update
```

Reply:
LGTM :-) thanks.